substrate-query-framework/index-builder/src/entities/SubstrateEventEntity.ts

```diff
@@ -55,6 +65,31 @@ export class SubstrateEventEntity {
   @JoinColumn()
   extrinsic?: SubstrateExtrinsicEntity;
 
+  // Warthog Fields
```

Comment:
We can move these fields to an abstract class `WarthogFields` and then extends our entities from this class.

Reply:
Done

Reply:
👍